src/options.rs

```diff
@@ -208,10 +192,65 @@ impl Orientation {
     }
 }
 
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+pub enum ColorInversion {
+    Normal,
+    Inverted,
+}
+
+impl Default for ColorInversion {
+    fn default() -> Self {
+        Self::Normal
+    }
+}
+
+impl From<bool> for ColorInversion {
+    fn from(val: bool) -> Self {
+        match val {
+            true => Self::Inverted,
+            false => Self::Normal,
+        }
+    }
+}
+
+impl DcsCommand for ColorInversion {
+    fn instruction(&self) -> Instruction {
+        match self {
+            Self::Normal => Instruction::INVOFF,
+            Self::Inverted => Instruction::INVON,
+        }
+    }
+
+    fn fill_params_buf(&self, _buffer: &mut [u8]) -> Result<usize, Error> {
+        Ok(0)
+    }
+}
+
```

Comment:
This is the only implementation of `DcsCommand` outside the `dcs` module and should probably be removed. `Invon` and `Invoff` are already contained in the `dcs` module.

Reply:
The reason why it's implemented is so inits can do `dcs.write_command(options.color_inversion)` instead of something more cumbersome. I'm not 100% sure happy with it though.

Reply:
I think it would be cleaner to move the `DcsCommand` impl to a new `SetInvertMode` command in the `dcs` module. `dcs.write_command(SetInvertMode(options.color_inversion))` is still relatively short and the explicit command also makes the code easier to understand IMO.

Reply:
fixed